scripts/checks.sh

```diff
@@ -62,18 +62,22 @@ function check_balenaOS()
 	# VERSION="1.24.0"
 	# PRETTY_NAME="Resin OS 1.24.0"
 	if grep -q -e '^VERSION="1.*$' -e '^PRETTY_NAME="Resin OS 1.*$' /etc/os-release; then
-		log_status "${BAD}" "${FUNCNAME[0]}" "resinOS 1.x is now completely deprecated"
+		log_status "${BAD}" "${FUNCNAME[0]}" "ResinOS 1.x is now completely deprecated"
 	else
 		# shellcheck disable=SC1091
 		source /etc/os-release
+		if [[ "${DEVICE_TYPE}" != "${SLUG}" ]]; then
```

Comment:
Where is `${DEVICE_TYPE}` coming from here

Reply:
In line 5 we source `/usr/sbin/resin-vars` which provides this envvar.

Reply:
hmm. I wasn't aware that `SLUG` in `etc/os-release` and `DEVICE_TYPE` in config.json are different..

Reply:
I think that's only true iff the device type is custom (`SLUG` indicating the base type, `DEVICE_TYPE` indicating the custom type). That is what's indicated in the attached issues, at least.

Reply:
I see. TIL